src/ProjectNeon/Assets/Features/Effects/FeedOnEffect.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+using System;
+
+public sealed class FeedOnEffect : ConditionalEffect
+{
+    private Effect _origin;
+    private string _attribute;
+
+    public FeedOnEffect(Effect origin, string attribute) : base(origin)
```

Comment:
Better to accept FeedType in the ctor. 

A secondary ctor can parse it, if desired. 

Otherwise, this object can be in an invalid state.